src/graspit_commander/graspit_commander.py

```diff
@@ -331,3 +353,70 @@ def dynamicAutoGraspComplete(id=0):
             return result.GraspComplete
         elif result.result is DynamicAutoGraspComplete._response_class.RESULT_INVALID_ID:
             raise InvalidRobotIDException(id)
+
+    @staticmethod
+    def importObstacle(obstacleName):
+        _wait_for_service('importObstacle')
+
+        serviceProxy = rospy.ServiceProxy('importObstacle', ImportObstacle)
+        result = serviceProxy(obstacleName)
+
+        if result.result is ImportObstacle._response_class.RESULT_SUCCESS:
+            return
+        elif result.result is ImportObstacle._response_class.RESULT_FAILURE:
+            raise ImportException("obstacle", obstacleName)
```

Comment:
Any reason you went with the generalized ImportException instead of more specific exceptions? 

I was thinking it might be better to have an exception defined for each 'type' and have them all extend from ImportException. That way you could specifically catch an ObstacleException or a RobotException, etc. which down the line may be useful when we've got more complex functions that work with multiple different imports in a single call. 

@jvarley what do you think. 

I'm not particularly tied to doing it either way, I just think in terms of making the code easier to expand in the future it may make more sense to have separate exceptions for each type of import for now.